Dockerfile

```diff
@@ -20,7 +20,8 @@ RUN apk add --no-cache -U \
 
 RUN pip install mcstatus yq
 
-HEALTHCHECK CMD mcstatus localhost:$SERVER_PORT ping
+ARG HEALTHCHECK_DEFAULT_START=5m
```

Comment:
Are you good with changing this to `1m` as a compromise for vanilla containers?

Reply:
that should work.

Reply:
Cool. When you change that I'll be good to merge this in.